fastlane/Fastfile

```diff
@@ -235,17 +242,43 @@ platform :ios do
   #
   desc 'Executes the initial steps needed during code freeze'
   lane :start_code_freeze do |options|
-    ios_codefreeze_prechecks(options)
+    ensure_git_status_clean
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+    ensure_git_branch(branch: DEFAULT_BRANCH)
+
+    confirmation_message = <<-MESSAGE
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/c13a215366b1c2abb0171201a62dc6a39267e30b/fastlane/Fastfile#L249"></span>Useless assignment to variable - <code>confirmation_message</code>.</td>
    </tr>
  </tbody>
</table>

Reply:
Fixed in https://github.com/Automattic/pocket-casts-ios/pull/1137/commits/d951f537ecef9223930535e1d1bc9bf9a14b97f8